gulpfile.js

```diff
@@ -2,5 +2,26 @@ const gulp = require('gulp')
 
 const pug2html = require('./gulp/tasks/pug2html')
 const styles = require('./gulp/tasks/styles')
+const img = require('./gulp/tasks/img')
 
-module.exports.start = gulp.series(pug2html, styles)
+const server = require('browser-sync').create()
+
+browserSync = module.exports = function serve(cb) {
```

Comment:
It looks very strange.

`module.exports` allows you to grant access to any module's entity from another one.

Just define all your functions and variables as usual and then export them:

```js
const browserSync = (cb) => { /*...*/ }

const startTask = gulp.series(/*...*/)

module.exports = {
  browserSync,
  startTask,
}
```

Reply:
You don't need to name anonymous function here because it already has a name – `browserSync`.